cmd/composectl/cmd/update/install.go

```diff
@@ -0,0 +1,113 @@
+package updatectl
+
+import (
+	"fmt"
+	"github.com/foundriesio/composeapp/pkg/compose"
+	v1 "github.com/foundriesio/composeapp/pkg/compose/v1"
+	"github.com/foundriesio/composeapp/pkg/update"
+	"github.com/schollz/progressbar/v3"
+	"github.com/spf13/cobra"
+)
+
+var (
+	installCmd = &cobra.Command{
+		Use:   "install",
+		Short: "Install the updated apps",
+		Long:  `Install the updated applications by extracting the compose project and loading its images into the docker store`,
+	}
+)
+
+type (
+	installOptions struct {
+	}
+
+	progressRendererCtx struct {
+		bar        *progressbar.ProgressBar
+		curImageID string
+		curLayerID string
+	}
+)
+
+func init() {
+	opts := installOptions{}
+
+	installCmd.Run = func(cmd *cobra.Command, args []string) {
+		installUpdateCmd(cmd, args, &opts)
+	}
+
+	UpdateCmd.AddCommand(installCmd)
+}
+
+func installUpdateCmd(cmd *cobra.Command, args []string, opts *installOptions) {
+	cfg, err := v1.NewDefaultConfig()
+	ExitIfNotNil(err)
+
+	updateCtl, err := update.GetCurrentUpdate(cfg)
+	ExitIfNotNil(err)
+
+	err = updateCtl.Install(cmd.Context(), compose.WithInstallProgress(getProgressRenderer()))
+	ExitIfNotNil(err)
+}
+
+func getProgressRenderer() compose.InstallProgressFunc {
+	ctx := &progressRendererCtx{}
+
+	return func(p *compose.InstallProgress) {
+		switch p.AppInstallState {
+		case compose.AppInstallStateComposeInstalling:
+			{
+				fmt.Printf("Installing app %s\n", p.AppID)
+			}
+		case compose.AppInstallStateComposeChecking:
+			{
+			}
```

Comment:
Did you leave this blank intentionally?

Reply:
Yes, this is kind of TODO or nice to implement.

Reply:
I think I better remove it and add TODO comment.

Reply:
https://github.com/foundriesio/composeapp/commit/3555aef8b6e2066f2a400f8dc85c8639ff24a280